payments/tests/test_views/test_gateway_options_list.py

```
@@ -0,0 +1,152 @@
+        self.client = APIClient()
+        self.url = reverse("payments:gateway-options-list")
+
+    def test_list_gateway_options_unauthenticated_success(self):
+        """Test that unauthenticated users can access the endpoint."""
+        GatewayOptionFactory(name="STRIPE")
+        GatewayOptionFactory(name="PAYPAL")
+
+        response = self.client.get(self.url)
+
+        assert response.status_code == status.HTTP_200_OK
+        assert len(response.data) == 2  # noqa: PLR2004
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)